### ISSUE TITLE
Show video length in list

### ISSUE THREAD

**Issue by pbalthasar** [agentcooper/Telik#15]
It would be awesome to have the length of any video in the list. 
Something like in the edited screenshot

<img width="569" alt="Bildschirmfoto 2024-04-11 um 19 51 07" src="https://github.com/agentcooper/Telik/assets/16125838/41386bd5-04f1-4a28-854b-c4f2fbf14419">

**Comment by agentcooper** [comment]
Agree, but this information is not available in RSS feeds (e.g. https://www.youtube.com/feeds/videos.xml?channel_id=UC3XTzVzaHQEd30rQbuvCtTQ) which Telik uses to fetch YouTube data.

**Comment by pbalthasar** [comment]
Ah I see, too bad. So my other issue isn't possible as well. Thanks for letting me know.
